server/game/cards/14-HCWM/PeteSpence.js

```diff
@@ -0,0 +1,55 @@
+const DudeCard = require('../../dudecard.js');
+const GameActions = require('../../GameActions/index.js');
+const StandardActions = require('../../PlayActions/StandardActions.js');
+
+class PeteSpence extends DudeCard {
+    setupCardAbilities(ability) {
+        this.job({
+            title: 'Pete Spence',
+            playType: 'noon',
+            cost: ability.costs.bootSelf(),
+            target: {
+                activePromptTitle: 'Mark an in-town location other than Pete\'s current location',
+                cardCondition: {
+                    location: 'play area',
+                    controller: 'any',
+                    condition: card => !card.isOutOfTown() && card.gamelocation !== this.gamelocation
+                },
+                cardType: ['location', 'townsquare', 'outfit']
```

Comment:
'location' is enough as it represents all locations including Town Square and outfits.
```suggestion
                cardType: ['location']
```

Reply:
:heavy_check_mark: